css/amp.css

```diff
@@ -287,6 +287,17 @@ template {
   display: none !important;
 }
 
+/**
+ * Provide common border-box sizing pattern to authors
```

Comment:
Please add slightly more comments here. E.g. "authors can set this class on <html> element to ..."